daemon/cluster/cluster.go

```diff
@@ -1367,7 +1363,7 @@ func (c *Cluster) CreateNetwork(s apitypes.NetworkCreateRequest) (string, error)
 	defer c.RUnlock()
 
 	if !c.isActiveManager() {
-		return "", c.errNoManager()
+		return "", errors.NewRequestNotAcceptableError(c.errNoManager())
```

Comment:
Not acceptable is used for accept headers that aren't satisfiable. A 421 might make sense here.